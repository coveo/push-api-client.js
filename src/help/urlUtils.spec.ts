// TODO: should be simple enough

// import {PushUrlBuilder} from './urlUtils';

// describe('UrlBuilder', () => {
//   describe('PushUrlBuilder', () => {
//     it('should return the base url', () => {
//       new PushUrlBuilder('my-source-id');
//     });
//   });
//   describe('StreamUrlBuilder', () => {});
// });
