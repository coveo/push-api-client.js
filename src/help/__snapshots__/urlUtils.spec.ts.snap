// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UrlBuilder PushUrlBuilder should return the base url 1`] = `"https://api.cloud.coveo.com/push/v1/organizations/org-id/sources/source-id"`;

exports[`UrlBuilder PushUrlBuilder should return the base url for update 1`] = `"https://api.cloud.coveo.com/push/v1/organizations/org-id/sources/source-id/documents/batch"`;

exports[`UrlBuilder PushUrlBuilder should return the file container url 1`] = `"https://api.cloud.coveo.com/push/v1/organizations/org-id/files"`;

exports[`UrlBuilder StreamUrlBuilder should return the base url 1`] = `"https://api.cloud.coveo.com/push/v1/organizations/org-id/sources/source-id/stream"`;

exports[`UrlBuilder StreamUrlBuilder should return the base url for update 1`] = `"https://api.cloud.coveo.com/push/v1/organizations/org-id/sources/source-id/stream/update"`;

exports[`UrlBuilder StreamUrlBuilder should return the file container url 1`] = `"https://api.cloud.coveo.com/push/v1/organizations/org-id/files"`;
